okhttp-clients/src/main/java/com/palantir/conjure/java/okhttp/RemotingOkHttpCall.java

```diff
@@ -151,6 +149,26 @@ public void onResponse(Call _call, Response response) {
         }
     }
 
+    private InterruptedIOException handleInterruption(SettableFuture<Response> future) {
```

Comment:
might consider `@CheckReturnValue` so we don't refactor and forget to throw

Reply:
Inlined instead